app/optscale_api/helpers/auth_tokens_dependency.py

```diff
@@ -0,0 +1,42 @@
+from __future__ import annotations
+
+import logging
+
+from app.core.exceptions import UserAccessTokenError
+from app.optscale_api.auth_api import OptScaleAuth
+
+logger = logging.getLogger("helper")
+
+
+TOKEN_ERROR_MESSAGE = "Failed to obtain access token for user ID: {}"
+
+
+def get_auth_client() -> OptScaleAuth:
+    return OptScaleAuth()
+
+
+async def get_user_access_token(
+    user_id: str, admin_api_key: str, auth_client: OptScaleAuth
+) -> str | Exception:
```

Comment:
Great for adding this, it's a useful function! But if you have created the whole `helpers` module just for it? I'd just put it in `auth_api` to make it easier to read and less intimidating for new maintainers.

Also the `auth_tokens_dependency` name got me confused intially as I thought this was a pydantic dependancy

Reply:
Actually now that I'm reading the function in more detail, I don't think we need this function at all, all it does is call `auth_client.obtain_user_auth_token_with_admin_api_key` with the same arguments and logs, something we can do already with the exception wrapper function you wrote above.